pkg/util/ready/ready.go

```diff
@@ -15,6 +15,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	appsv1client "k8s.io/client-go/kubernetes/typed/apps/v1"
 	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"github.com/Azure/ARO-RP/pkg/util/version"
```

Comment:
You will need to put a line break here -- run `make validate-go` and it'll fix it or warn you if there's code formatting errors.